packages/code-du-travail-frontend/src/conventions/Search/Form.js

```diff
@@ -222,8 +224,18 @@ const ResultsContainer = styled.div`
 `;
 
 const ConventionLink = styled.a`
-  color: ${colors.altText};
+  display: inline;
```

Comment:
Nico a rajouté une variante pour avoir des boutons qui ont le même look & feel qu'un lien ici https://socialgouv-react-ui.netlify.com/?path=/story/components-button--link. C'est juste pas la même couleur mais pour le reste, it's a match, ça permettrait d'enlever queleques styles ici !

Reply:
je fais le test ! merci

Reply:
bon en fait ca marche pas, car le style ConventionLink est aussi a utilisé dans le bouton
Ce qui donne `<Button variant="navLink" as={ConventionLink} />` or convention link étant un lien on revient au pb initial, c'est pas navigable au clavier. 

Il faudrait faire l'inverse `<ConventionLink as={ConventionButton}` mais du coup on rajoute un nouveau styledComponent...

Reply:
Ah oui ? Je pensais  à truc comme un `<ConventionButton>` qui serait un `styled(Button)` avec la variant color qu'il faut mais je rate peut-etre quelque chose

Reply:
Ah je re-teste !